### PR TITLE
OTG: Unique name for device not to overlap with Lag name

### LINKED ISSUE #558: OTG: RT-5.2: Aggregate Interfaces
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/interface/aggregate/otg_tests/aggregate_test/README.md)

### LINKED ISSUE #559: OTG: RT-5.3: Aggregate Balancing
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/interface/aggregate/otg_tests/balancing_test/README.md)

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
Unique name for device not to overlap with Lag name.

resolves #558
resolves #559

### CODE REVIEW

File: feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go
Comment (jasdeep-hundal) [inline feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go:1]: The ATE test always shuts down ports (as opposed to disabling LACP members), please remove https://github.com/openconfig/featureprofiles/pull/2007/files#diff-dd661b744b84b3df218ef0ba5014877ee827c8a630ed6a70f82d512248ca058dR467-R488 and always do a port shutdown for both the static LAG and LACP test cases.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: proto/metadata_go_proto/metadata.pb.go
Comment (jasdeep-hundal) [inline proto/metadata_go_proto/metadata.pb.go:1]: Please run goimports on this file to order the imports correctly.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: proto/metadata_go_proto/metadata.pb.go
Comment (octpetre) [reply proto/metadata_go_proto/metadata.pb.go:1]: Done

File: feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go
Comment (octpetre) [reply feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go:1]: Done

**Review by jasdeep-hundal — APPROVED** [review]

**Pull request merged**